Dev-Corner/slides/tool_integration.html

```diff
@@ -45,65 +45,1274 @@
+                            <https://bitbucket.org/nsegata/graphlan/overview>`_.
+
+                            ]]></help>
+                            ```
+
+                            <small>[reStructuredText markdown format](http://docutils.sourceforge.net/docs/user/rst/quickref.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### `citations`
+
+                            <img src="../images/citations.png" alt="citations"/>
+
+                            ```
+                            <citations>
+                                <citation type="doi">10.1093/bioinformatics/bts611</citation>
+                                <citation type="doi">10.1093/nar/gks1219</citation>
+                                <citation type="doi">10.1093/nar/gks1005</citation>
+                                <citation type="doi">10.1093/bioinformatics/btq461</citation>
+                                <citation type="doi">10.1038/nbt.2198</citation>
+                            </citations>
+                            ```
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Wrapper directory tree
+
+                            ```
+                            graphlan/
+                            ├── graphlan.xml
+                            ├── test_data/
+                                ├── input_tree.txt
+                                ├── png_image.png
+
+                            ```
+
+                            [Details of XML tag sets](https://wiki.galaxyproject.org/Admin/Tools/ToolConfigSyntax)
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/planemo-logo.png" alt="Planemo" class="no-border" height=25/>
+                            #### Building Galaxy Tools
+
+                            <img src="../images/planemo_tool_building_lint.png" alt="Building tools" class="no-border"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            <img src="../images/exercise.png" alt="exercise"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### The Goal
+                            A wrapper for Seqtk, a tool for processing sequence data in FASTA and FASTQ files.
```

Comment:
I would have moved this slide and the next one right after the slide with the tutorial objectives, to introduce the hands on use during all the tutorial. What do you think?